crates/bevy_render/src/renderer/mod.rs

```diff
@@ -355,6 +355,7 @@ pub async fn initialize_renderer(
                 label: options.device_label.as_ref().map(AsRef::as_ref),
                 required_features: features,
                 required_limits: limits,
+                memory_hints: wgpu::MemoryHints::default(),
```

Comment:
Probably want to expose this in the render plugin settings like we do with other things.